### DISCUSSION TITLE
ubuntu lts debian import freeze 2022-02-24

### DISCUSSION THREAD

**Discussion by dankamongmen** [dankamongmen/notcurses#2621] [Q&A]
the next ubuntu LTS will be "[jammy jellyfish](https://discourse.ubuntu.com/t/jammy-jellyfish-release-schedule/23906)" 22.04. its debian import freeze is in about two weeks. we feed ubuntu through debian, so that's our last chance to get updates into this LTS.

what's the most important thing to resolve? i'm personally thinking it's CLI mode scrolling, but i'd be interested in hearing other opinions.

**Comment by michaelsbradleyjr** [comment]
CLI mode scrolling does seem like an important thing to resolve, though in #2414 you noted it's "going to define 4.0, i think".

Maybe there's lower-hanging fruit with Feb 24 coming up soon?
